examples/vscode-sandbox/README.md

```diff
@@ -98,19 +98,47 @@ $ kubectl wait --for=condition=Ready sandbox sandbox-example
 $ kubectl get pods -o jsonpath=$'{range .items[*]}{.metadata.name}: {.spec.runtimeClassName}\n{end}'
 ```
 
-## Accesing vscode
+## Accessing VSCode
 
-Port forward the vscode server port.
+To access the VSCode instance securely and reliably, use the Sandbox Router.
 
+### Option 1: Production (via Gateway)
+
+If you have deployed the Gateway **and** Router (see `clients/python/agentic-sandbox-client/sandbox_router`), access VSCode via the external Gateway IP.
+
+1. Get the Gateway IP:
+```bash
+export GATEWAY_IP=$(kubectl get gateway external-http-gateway -n default -o jsonpath='{.status.addresses[0].value}')
+echo "Gateway IP: $GATEWAY_IP"
+```
+
+2. **Connect:** You must inject the routing headers. Use `curl` to test:
+```bash
+# Replace <GATEWAY_IP> with the actual IP
+curl -v -H "X-Sandbox-ID: sandbox-example" \
```

Comment:
has this been tested ?  accessing the vscode UI via router ?

Reply:
Yeah i can add instructions for the UI as well 
<img width="3424" height="1594" alt="image" src="https://github.com/user-attachments/assets/604c3a5f-c9ad-4653-8933-96033a45e050" />